src/greeter/GreeterApp.h

```diff
@@ -65,6 +68,7 @@ namespace SDDM {
         UserModel *m_userModel { nullptr };
         GreeterProxy *m_proxy { nullptr };
         KeyboardModel *m_keyboard { nullptr };
+        QStringList m_pamMessages { };
```

Comment:
just QStringList
The others need it to initialise a pointer, that's not relevant here.